pkg/config/changefeed.go

```diff
@@ -202,7 +202,8 @@ type ChangefeedConfig struct {
 	Epoch   uint64 `json:"epoch"`
 	BDRMode bool   `json:"bdr_mode" default:"false"`
 	// redo releated
-	Consistent *ConsistentConfig `toml:"consistent" json:"consistent,omitempty"`
+	Consistent *ConsistentConfig          `toml:"consistent" json:"consistent,omitempty"`
+	Scheduler  *ChangefeedSchedulerConfig `toml:"scheduler" json:"scheduler,omitempty"`
```

Comment:
Could we just add EnableTableAcrossNodes in ChangefeedConfig, instead of whole Scheduler